### PR TITLE
need to rebase again

### PR DESCRIPTION
**Pull request by queryluke**

### CODE REVIEW

**Comment by sdellis** [comment]
@roastduckalamode I'm confused by this flow.  I assumed that we would always be merging into gh-pages from the master branch, not the other way around.  All PRs should be merged into master and then master into gh-pages, which I usually do in quick succession so we don't have lingering changes that haven't been "published" to the site.  Let me know if I am missing something!

**Comment by queryluke** [comment]
@sdellis That workflow works as well, but then gh-pages is multiple commits ahead of master. Which  makes gh-pages and master "out of sync" commit-wise (even though the code is the same). 

I updated the readme last week with how I'm processing the changes. It's a workflow that I found online. it uses `git rebase master` on the gh-pages branch which keeps them in sync. I feel this keeps true to the github idea of a master branch, whereas the other way, we're treating master as just a staging branch for gh-pages.

In short, I pull down the branch that has a pull request and check it locally. If it's ok, I merge it with master using the github website gui. This allows me to make comments & notes.

Then locally using CLI I pull origin/master ->  checkout gh-pages ->  rebase master -> check that I've nto broken anything -> push origin gh-pages

Does that make sense?

**Comment by sdellis** [comment]
Yes, that works!  :+1:  I just want to make sure we're all on the same page.  I either missed your announcement to the group, or we should make one so folks know what the correct procedure is.  There are many possible gitflows, and I think your approach makes sense for us.
